src/main/java/uk/gov/companieshouse/confirmationstatementapi/eligibility/EligibilityFailureReason.java

```diff
@@ -5,5 +5,6 @@
     INVALID_COMPANY_STATUS,
     INVALID_COMPANY_TYPE_CS01_FILING_NOT_REQUIRED,
     INVALID_COMPANY_TYPE_PAPER_FILING_ONLY,
-    INVALID_COMPANY_TYPE_USE_WEB_FILING
+    INVALID_COMPANY_TYPE_USE_WEB_FILING,
+    INVALID_OFFICER_COUNT
```

Comment:
Similar to this comment [here](https://github.com/companieshouse/confirmation-statement-api/pull/27#discussion_r637736958)
I'd reccommend being more descriptive. Perhaps: `INVALID_COMPANY_APPOINTMENTS_MORE_THAN_ONE_OFFICER`
So a user only interacting with the api has more of an idea what is wrong with the officer count

Reply:
I took INVALID_OFFICER_COUNT from the Validation errors enum in the schema.json shall i replace that with the more descriptive version or add this as a new entry and leave that alone ?

Reply:
yes best to replace it with the more descriptive one.
I added INVALID_OFFICER_COUNT way back when there was one type of validation on officers. Now with multiple types, i don't think we'd ever use the more generic "INALID_OFFICER_COUNT"